apps/web/src/services/asset-inventory/collector/shared/RecentCollectorJobList.vue

```diff
@@ -46,6 +48,15 @@ const state = reactive({
     }),
 });
 
+const handleTooltipContent = (job: MinimalJobInfo) => {
+    const { status, finished_at } = job;
+    let content = '';
+    if (status === JOB_STATE.SUCCESS) content = 'INVENTORY.COLLECTOR.MAIN.JOB_SUCCESS';
+    if (status === JOB_STATE.CANCELED) content = 'INVENTORY.COLLECTOR.MAIN.JOB_CANCELED';
+    if (status === JOB_STATE.FAILURE) content = 'INVENTORY.COLLECTOR.MAIN.JOB_FAILURE';
+    return i18n.t(content, { date: dayjs.utc(finished_at).tz(storeState.timezone).format('YYYY-MM-DD hh:mm:ss') });
+};
```

Comment:
question: it works when users change the language at GNB?

Reply:
yup! it works!